### DISCUSSION TITLE
enable/disable rich presence based on a tag/list file

### DISCUSSION THREAD

**Discussion by siloricity** [s0hv/foo_discord_rich#15] [Ideas]
there are some songs/audio that i would like to keep hidden from the status and i dont want to manually disable each time

**Comment by s0hv** [comment]
I don't see this as a common feature so I won't implement it. However, you can achieve something similar by just conditionally removing all text from the status so it only says the time. Using a custom metadata property it can be done like this `$if([%hide_drp%], ,[%title%])`
